.github/actions/setup-nix/action.yaml

```diff
@@ -0,0 +1,21 @@
+name: Setup Nix
+
+description: Installs nix, sets up cachix and installs a subset of tooling.
+
+inputs:
+  authToken:
+    description: Token to pass to cachix
+  tools:
+    description: Tools to install with nix-env -iA <tools>
+
+runs:
+  using: composite
+  steps:
+    - uses: cachix/install-nix-action@v13
```

Comment:
This might be a good opportunity to upgrade to https://github.com/cachix/install-nix-action/releases/tag/v16

Reply:
Yeah, good point. Did so now. Unfortunately I still can't run the workflow locally with `act`, still breaking at the `install-nix-action` step. Probably need to update to latest master of `act` at some point.

Updated the other actions at the same time, which allowed me to remove one of the parameters to the matrix.